common/ceph_release_info

```diff
@@ -13,5 +13,6 @@ export -A APP_RELEASE_NAMES=( [firefly]=icehouse
                               [octopus]=ussuri
                               [pacific]=wallaby # xena and wallaby both have pacific
                               [quincy]=yoga
-                              [reef]=bobcat )   # bobcat and caracal both have reef
+                              [reef]=bobcat     # bobcat and caracal both have reef
```

Comment:
the comment needs fixing since caracal will now not have reef

Reply:
done